src/blob_gc_job_test.cc

```diff
@@ -41,7 +41,7 @@ class BlobGCJobTest : public testing::Test {
     options_.env->CreateDirIfMissing(dbname_);
     options_.env->CreateDirIfMissing(options_.dirname);
   }
-  ~BlobGCJobTest() {}
+  ~BlobGCJobTest() { Close(); }
```

Comment:
Should be DestroyDB?

Reply:
I change all Destroy to Close in consistency with other unit tests we have.